lib/node_modules/@stdlib/stats/base/svariancepn/src/main.c

```diff
@@ -53,25 +69,20 @@ float stdlib_strided_svariancepn( const int64_t N, const float correction, const
 	if ( N <= 0 || n <= 0.0f ) {
 		return 0.0f / 0.0f; // NaN
 	}
-	if ( N == 1 || stride == 0 ) {
+	if ( N == 1 || strideX == 0 ) {
 		return 0.0f;
 	}
-	// Compute an estimate for the mean:
-	mu = (double)stdlib_strided_ssumpw( N, X, stride ) / dN;
+	mu = API_SUFFIX(stdlib_strided_ssumpw_ndarray)( N, X, strideX, offsetX ) / dN;
 
-	if ( stride < 0 ) {
-		ix = (1-N) * stride;
-	} else {
-		ix = 0;
-	}
+	ix = offsetX;
 	// Compute the variance...
```

Comment:
```suggestion

	// Compute the variance...
```

Follow our empty line before a comment convention.